qwik-graphql-tailwind/src/routes/auth/redirect/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { component$, useClientEffect$ } from '@builder.io/qwik';
+import type { DocumentHead } from '@builder.io/qwik-city';
+import { AUTH_TOKEN, GET_TOKEN_URL } from '~/utils/constants';
+import * as styles from '../signin/signin.classNames';
+
+export default component$(() => {
+  useClientEffect$(async () => {
+    const abortController = new AbortController();
+    const data = await getAuthToken(abortController);
+    sessionStorage.setItem(AUTH_TOKEN, data.access_token);
+    window.location.href = '/';
```

Comment:
Using `window.location.href` feels hacky, could we use [redirect](https://qwik.builder.io/qwikcity/data/redirects/) instead?

Reply:
since this is handled in the browser, there is no way to make use of qwik redirect here.